smartmin/models.py

```diff
@@ -193,10 +194,17 @@ def import_xls(cls, filename, user, import_params, log=None, import_results=None
                 try:
                     field_values = cls.prepare_fields(field_values, import_params, user)
                     record = cls.create_instance(field_values)
+
+                    record_error_description = ''
+                    if isinstance(record, tuple):
```

Comment:
I think this would be clearer as a particular exception type raised in `create_instance`